app/settings/config.py

```diff
@@ -1,7 +1,15 @@
+import pathlib
+
 from pydantic import Field
 from pydantic_settings import BaseSettings
 
+BASE_PATH = pathlib.Path(__file__).resolve().parent.parent  # app folder
+
 
 class Config(BaseSettings):
     ms_teams_webhook: str = Field(alias='webhook')
     posting_interval: int = 60
+    imgur_client_id: str = Field(alias='imgur_client_id')
+    path_to_toad_links: pathlib.Path = (
+        BASE_PATH / 'infra/repositories/picture/toad_memes.json'
```

Comment:
Why do you construct a link like this?

Reply:
I like this solution, because it saves relativity (you set path to file relative to you base folder (`app` inside which all code located), like this: `infra/repos/.../.../some_file.json` (similar to all imports in my code), and because of relativity, code will not break if it will be moved to someone else computer. But at the same time it is clear, where this file is located, which is not clear if I would use relative path like `../../../some_file.json` inside code).

Reply:
Would this solution work if you try to debug it on Windows? Paths work a bit different way on other OS

Reply:
Yes, it will work, because pathlib smart enough.
![image](https://github.com/imPDA/CheerUpYourColleagues/assets/43822452/33021550-34bc-4fcd-a79a-9fd77ebe8d2b)
![image](https://github.com/imPDA/CheerUpYourColleagues/assets/43822452/0366c8d4-687d-4ee9-820c-f5e476fdbfd0)